docs/api/hooks.md

```diff
@@ -68,16 +67,71 @@ export const CounterComponent = () => {
 Using props to determine what to extract:
 
 ```jsx
-import React from 'react'
+import React, {useCallback} from 'react'
 import { useSelector } from 'react-redux'
 
-export const TodoListItem = props => (
-  const todo = useSelector(state => state.todos[props.id], [props.id])
+export const TodoListItem = ({id}) => (
+  const selector = useCallback(state => state.todos[id], [id])
+  const todo = useSelector(selector)
 
   return <div>{todo.text}</div>
 }
 ```
 
+With Reselect:
+
+```jsx
+// ducks/todos.js
+import { createSelector } from 'reselect'
+
+export const selectPendingTodos = createSelector(
+  (state) => state.todos,
+  (todos) => todos.filter(todo => todo.completed !== true)
+)
+
+export const selectNPendingTodos = createSelector(
```

Comment:
I am a big fan of proper naming, so I would make sure the examples do it to, i.e. call this `selectNrOfPendingTodos`